route/build/src/mpi_process.f90

```diff
@@ -1259,6 +1306,7 @@ SUBROUTINE mpi_route(pid,           & ! input: proc id
                   basinRunoff_trib,  &  ! input: basin (i.e.,HRU) runoff (m/s)
                   basinEvapo_trib,   &  ! input: basin (i.e. HRU) Evapo  (m/s)
                   basinPrecip_trib,  &  ! input: basin (i.e. HRU) Precip (m/s)
+                  basinCC_trib,      &  ! input: basin (i.e. HRU) constituent (g)
```

Comment:
Again maybe tie in solute.

Reply:
change `basinCC_trib` to `basinSolute_trib`